_docs/ioos-metadata-profile-v1-2.md

```diff
@@ -433,7 +434,7 @@ In partnership with IOOS, NOAA [NDBC](https://www.ndbc.noaa.gov/) ingests nonfed
 | Eh |   |
 
 NDBC will leverage IOOS RA ERDDAP servers as the central access points for nonfederal IOOS partner data.  This is a departure from the previous data delivery method, where providers and RAs packaged XML files and posted them to an NDBC FTP server. It is critical that RAs work with their regional data providers to ensure their data are available in the RA ERDDAPs, so that NDBC can continue to access those datasets. 
-![Data flow](images/NDBC-data-flow.png?raw=true)
+![Data flow](NDBC-data-flow.png)
```

Comment:
I forgot that this is a jekyll page so the images are copied to the root of the page.